docs/understand/versioning-internals.md

```diff
@@ -98,3 +98,33 @@ Both these types of metadata are not only mutable, but also require strong consi
 Luckily, this is also much smaller set of metadata compared to the committed metadata.
 
 References and uncommitted metadata are currently stored on a key-value store (See [supported databases](../reference/configuration.md)) for consistency guarantees.
+
+## Merging changes to data
+
```

Comment:
It's best to add a sentence about using merge and user control over it - `For merge, you need to specify what is the source and what is the destiny`

Reply:
Added